backend.native/tests/build.gradle

```diff
@@ -2036,6 +2036,12 @@ task extend_exception(type: RunKonanTest) {
     source = "runtime/exceptions/extend0.kt"
 }
 
+task custom_hook(type: RunStandaloneKonanTest) {
```

Comment:
Testing Swift -> Kotlin call case may also be useful.

Reply:
Manually tested.